app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,29 @@
+class PostsController < ApplicationController
+  require "will_paginate/array"
+  PAGE_SIZE = 4
+
+  def index
+    page = (params[:page] || '1').to_i
+
+    client = Tumblr::Client.new({
```

Comment:
Yeah `static` was the wrong word there. I guess non-programmatic. We're talking about the content that any normal business would put on their website, like their address and the names of their employees. Everything that doesn't have to do with the temperature monitoring app.